src/de.html

```diff
@@ -462,8 +467,9 @@ <h3 class="header" itemprop="name">Welche sind die laufenden Gebühren?</h3>
             </div>
             <div class="content">
               <p>Leider ist Ihr Browser zu alt und kann die kryptographische mathematische Operation nicht bearbeiten, die erforderlich ist, um Ihren QR-Code zu dekodieren.</p>
-              <p>Für iOS-Nutzer ist iOS 14 erforderlich. Wenn Sie ein iPhone ab iPhone6S haben, müssen Sie iOS aktualisieren, um die App nutzen zu können.</p>
-              <p>Wenn Sie nicht aktualisieren können oder nicht möchten, können Sie diese Website auf Ihrem Computer öffnen und ihren Browser verwenden um Ihre Karte zu generieren.</p>
+              <p>For iOS users, iOS 14 is required. iPhone6S and up are supported but you need to upgrade iOS to use the app.</p>
```

Comment:
Can be condidered as a regression here but the content changed a bit so ... I gonna let it pass.